info.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "factorio-world",
-  "version": "2.0.1",
+  "version": "2.0.2",
```

Comment:
Oh actually only nitpick here, this feels more like at least a version 2.1, the last digit I'd reserve just for bug fixes or other tiny changes. New features, when backwards compatible, should be a minor version change at least.

Since everything is backwards compatible it's OK not to make this a version 3.

Reply:
ya sure. I can change it.  

My mods are using 2.0.x as experimental releases.  That habit carried over.